bundle-workflow/tests/test_integ_workflow/integ_test/test_integ_test_suite.py

```diff
@@ -0,0 +1,169 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import unittest
+from unittest.mock import call, patch
+
+from git.git_repository import GitRepository
+from manifests.build_manifest import BuildManifest
+from manifests.bundle_manifest import BundleManifest
+from manifests.test_manifest import TestManifest
+from test_workflow.integ_test.integ_test_suite import (DependencyInstaller,
+                                                       IntegTestSuite,
+                                                       InvalidTestConfigError,
+                                                       MissingArtifactError,
+                                                       ScriptFinder)
+
+
+@patch("os.makedirs")
+@patch("os.chdir")
+@patch.object(GitRepository, "__checkout__")
+class TestIntegSuite(unittest.TestCase):
+    def setUp(self):
+        os.chdir(os.path.dirname(__file__))
+        with open("data/bundle_manifest.yml", "r") as file:
```

Comment:
You can use from_path and no need to do open. Something like https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/tests/tests_manifests/test_build_manifest.py#L21